src/Database/Schema/CachedCollection.php

```diff
@@ -20,6 +20,11 @@
 
 /**
  * Decorates a schema collection and adds caching
+ *
+ * @method array<string,mixed> listTablesAndViews() Get the list of tables available in the current connection.
+ * This will include any views in the schema.
+ * @method array<string,mixed> listTablesWithoutViews() Get the list of tables available in the current connection.
+ * This will exclude any views in the schema.
```

Comment:
You shouldn't need these annotations on this class as the methods are implemented.